nginx/etc/nginx/nginx.conf.ctmpl

```diff
@@ -1,107 +0,0 @@
-user  nginx;
```

Comment:
After `autopilotpattern/nginx:1-r6.1.0` the config files were moved around. See new conf files added elsewhere.